subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/fixture/CrossVersionPerformanceTestRunner.groovy

```diff
@@ -106,6 +108,13 @@ class CrossVersionPerformanceTestRunner extends PerformanceTestSpec {
         return measuredBuildOperations
     }
 
+    void loadConfiguredTestProject() {
+        def testProjectFromSystemProperty = System.getProperty(TEST_PROJECT_PROPERTY_NAME)
+        if (testProjectFromSystemProperty != null && !testProjectFromSystemProperty.empty) {
+            testProject = testProjectFromSystemProperty
+        }
+    }
```

Comment:
Given that this method is called from the constructor, can we make it `static` and let it return the state to be set?

Reply:
Done.